Swiftfin/Views/AdminDashboardView/ActiveSessions/ActiveSessionsView/ActiveSessionsView.swift

```diff
@@ -117,33 +117,33 @@ struct ActiveSessionsView: View {
             .tag(nil as Int?)
 
             Label(
-                300.formatted(.hourMinute),
+                Duration.seconds(300).formatted(.units(allowed: [.hours, .minutes])),
```

Comment:
Make an `hourMinuteAbbreviated` format style with the other `Duration.UnitsFormatStyle`s in `FormatStyle.swift`.